scss/tech.scss

```diff
@@ -1,31 +1,28 @@
+.tech-container {
+  display: grid;
+  grid-template-columns: repeat(3, 200px);
+  grid-template-rows: repeat(2, 300px);
+  padding-left: 20px;
+  font-family: 'Roboto', sans-serif;
+  justify-content: center;
+  grid-gap: 30px;
+}
+
 .tech {
   align-items: center;
-  background-color: $green;
-  border-radius: 5px;
   display: flex;
   flex-direction: column;
-  font-family: 'Roboto', sans-serif;
-  height: 300px;
-  justify-content: space-around;
-  width: 200px;
+  justify-content: center;
   position: relative;
+  width: 200px;
 
-  &::before {
-    border: 1px solid $green;
-    border-radius: 5px;
-    content: '';
-    height: inherit;
-    left: -10px;
-    position: absolute;
-    top: -10px;
-    width: inherit;
-    transition: top 0.5s, left 0.5s;
-    z-index: -1;
-  }
-
-  &:hover::before {
-    top: 10px;
-    left: 10px;
+  &:first-child {
+    grid-row: 1/3;
+    grid-column: 1/2;
+    border-right: 2px solid $green;
+    justify-self: center;
+    padding: 0 10px;
+    height: 100%;
```

Comment:
Define a min-height is better than using 100% as if it is shorter it would look bad.